docs/protocol/extension_statistics.md

```diff
@@ -111,8 +112,16 @@ $model_stat =
   DOES NOT include cache hits.
 
 - "inference_stats" : The aggregate statistics for the
-  model/version. So, for example, "inference_stats":"success"
-  indicates the number of successful inference requests for the model.
+  model. So, for example, "inference_stats":"success" indicates the number of
+  successful inference requests for the model.
+
+- "response_stats" : The aggregate decoupled response statistics for the
```

Comment:
```suggestion
- "response_stats" : The aggregate response statistics for the
```

Reply:
would replace mentions of "decoupled response" to just "response"

Reply:
Sure, they are all updated on `docs/protocol/extension_statistics.md`.